index.html

```diff
@@ -9,6 +9,50 @@
 </head>
 <body>
     <main>
+        <section id="popup-window">
+            
+            <div class="modal" id="modal">
+             <div id="projectDetailsPopup" class="popup">
+                 <span class="close-button" id="close-popup">&times;</span>
+                 <h4 id="popupName"></h4>
+                 <ul class="list">
+                     <li class="canopy" id="client"></li>
+                     <li class="bullet" id="role"></li>                        
+                     <li class="bullet" id="year"></li>
+                 </ul>
+                 <img id="popupImage" src="" alt="projectImage">
+                 <div class="flexPop">
+                    <div class="con1">
+                        <p id="popupDescription"></p>
+                     </div>
+                     <div class="con2">
+                        <ul class="buttonList" id="techList">
+                            <li><button class="smallbtn" id="tech1"></button> </li>
+                            <li><button class="smallbtn" id="tech2"></button> </li>
+                            <li><button class="smallbtn" id="tech3"></button> </li>
+                        </ul>
+                        <hr id="bottomLine">
```

Comment:
- [ ] This isn't a biggie but to ensure the best consistency with the Figma design I'd suggest you change the color of the horizontal line here to `#F7F7F9` as it's creating a discrepancy with the Figma design. 
- Figma design:
  ![image](https://github.com/Porkalaikarthika/Portfolio/assets/104968236/3063efba-84a7-494e-916b-132ab6ef9259)
- Your design:
  ![image](https://github.com/Porkalaikarthika/Portfolio/assets/104968236/9ccc6acc-a114-4473-9b3a-ddd32867d980)